src/schemas/686c-674/schema.js

```diff
@@ -0,0 +1,941 @@
+          type: 'string',
+        },
+      },
+    },
+  },
+  properties: {
+    optionSelection: {
+      type: 'object',
+      'view:selectable686Options': {
+        type: 'object',
+        properties: {
+          addChild: {type: 'boolean', default: false},
+          addSpouse: {type: 'boolean', default: false},
+          reportDivorce: {type: 'boolean', default: false},
+          reportDeath: {type: 'boolean', default: false},
+          reportStepchildNotInHousehold: {type: 'boolean', default: false},
+          reportMarriageOfChildUnder18: {type: 'boolean', default: false},
+          reportChild18orOlderIsNotAttendingSchool: {
+            type: 'boolean',
+            default: false,
+          },
+          report674: {
+            type: 'boolean',
+            default: false,
+          },
+        },
+      },
+    },
+
+    veteranInformation: {
+      type: 'object',
+      properties: {
+        veteranInformation: {
+          type: 'object',
+          properties: {
+            first: {
+              $ref: '#/definitions/genericTextInput',
+            },
+            middle: {
+              $ref: '#/definitions/genericTextInput',
+            },
+            last: {
+              $ref: '#/definitions/genericTextInput',
+            },
+            suffix: {
+              type: 'string',
+              enum: suffixes,
+            },
+            ssn: {
+              $ref: '#/definitions/genericNumberAndDashInput',
+            },
+            vaFileNumber: {
+              $ref: '#/definitions/genericNumberAndDashInput',
+            },
+            serviceNumber: {
+              $ref: '#/definitions/genericNumberAndDashInput',
+            },
+            birthDate: {
+              $ref: '#/definitions/date',
+            },
+          },
+        },
+      },
+      veteranAddress: {
+        type: 'object',
+        properties: {
+          veteranAddress: {
+            $ref: '#/definitions/addressSchema',
+          },
+          phoneNumber: {
+            $ref: '#/definitions/phoneInput',
+          },
+          emailAddress: {
+            $ref: '#/definitions/emailInput',
+          },
+        },
+      },
+    },
+
+    addChild: {
+      type: 'object',
+      properties: {
+        addChildInformation: {
+          type: 'object',
+          properties: {
+            childrenToAdd: {
+              type: 'array',
+              minItems: 1,
+              items: {
+                type: 'object',
+                properties: {
+                  first: {
+                    $ref: '#/definitions/genericTextInput',
+                  },
+                  middle: {
+                    $ref: '#/definitions/genericTextInput',
+                  },
+                  last: {
+                    $ref: '#/definitions/genericTextInput',
+                  },
+                  suffix: {
+                    type: 'string',
+                    enum: suffixes,
+                  },
```

Comment:
Should this use the imported `fullName` definition?

Reply:
There are quite a few changes like this that I ignored simply because it is more of a refactor, we should do this in a separate ticket.